nixops_encrypted_links/lib.py

```diff
@@ -58,7 +52,7 @@ def get_type(cls):
 
 
 def index_to_private_ip(index: int) -> str:
-    n = 105 + index / 256
+    n = 105 + index // 256
```

Comment:
The single `/` operator worked in python2 as a floor, but in python3 doesn't yield the quotient -- python3 needs the `//` operator.